docs/2023/Sunbird ED/Implement Independent SSO Flow on NPM Package/index.md

```diff
@@ -2,7 +2,10 @@
 sidebar_position: 1
 title: Introduction
 slug:/2023/Sunbird ED/Implement Independent SSO Flow on NPM Package
+<<<<<<< HEAD
+=======
 
+>>>>>>> b2616c721fbf87737df975d7627f70c90c35dd23
 ---
```

Comment:
These are the merge conflicts that you need to solve.
If you are unfamiliar with it, you can refer to this [guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)